Doc/library/importlib.rst

```diff
@@ -810,10 +810,71 @@ ABC hierarchy::
 
 .. class:: Traversable
 
-    An object with a subset of pathlib.Path methods suitable for
-    traversing directories and opening files.
+   An object with a subset of pathlib.Path methods suitable for
+   traversing directories and opening files.
 
-    .. versionadded:: 3.9
+   .. versionadded:: 3.9
+
+   .. abstractmethod:: name()
```

Comment:
I'm inclined to say let's not duplicate the docs from the code. Let's just link to the code if needed.